src/assets/i18n/en.json

```diff
@@ -0,0 +1,9 @@
+{
+  "language": {
+    "english": "EN",
+    "spanish": "ES"
+  },
+  "home": {
+    "init": "Init"
```

Comment:
This should be "home": "Home" or something like that. We use this notation: Inicio = Home

Reply:
Or maybe "start": "Home"